gh-weblog-2/content/posts/markdown/2014-09-20-01-21-59.md

```diff
@@ -1,4 +1,4 @@
-You run an open source project, and I have a meaningful contribution for you. I'd love to submit it, but now you're asking me for my personal information, without ever having shown that you can be trusted with that: I'm not signing your CLA.
+You run an open source project, and I have a meaningful contribution for you. I'd love to submit it, but now you're asking me for my personal information, without ever having shown that you can be trusted with that: I'm not signing your CLA (Contributor License Agreement).
```

Comment:
I think I'd like to keep this as just CLA. If that raises questions, and someone has to google it, that's the kind of exercise that has meaning for the person doing the looking-up.